src/components/we-are-using/index.js

```diff
@@ -247,6 +247,11 @@ const companies = [
 		name: 'Aarogya Setu',
 		href: 'https://www.aarogyasetu.gov.in/',
 		logo: 'aarogyasetu.svg'
+	},
+	{
+		name: 'RailRecipe',
+		href: 'https://www.railrecipe.com',
+		logo: 'railrecipe-logo.png'
```

Comment:
I think you might have forgotten to commit the logo itself 😬 Can't find a file called `railrecipe-logo.png` in the changeset of this PR.